components/shared/DownloadMapData.vue

```diff
@@ -350,9 +350,5 @@ const downloadKMLSelection = () => {
 </template>
 
 <style scoped>
-.button-container {
-  display: flex;
-  justify-content: center;
-  margin-top: 20px;
-}
+/* Remove the old button-container styles as we're using Tailwind now */
```

Comment:
:+1: on the work but I think we can omit the comment (and therefore style tag altogether).